javaparser-core/src/main/java/com/github/javaparser/printer/ConcreteSyntaxModel.java

```diff
@@ -64,17 +65,17 @@ private static CsmElement memberAnnotations() {
     }
 
     private static CsmElement annotations() {
-        return list(ObservableProperty.ANNOTATIONS, none(), none(), newline());
+        return list(ObservableProperty.ANNOTATIONS, space(), none(), newline());
     }
 
     private static CsmElement typeParameters() {
-        return list(ObservableProperty.TYPE_PARAMETERS, CsmElement.sequence(CsmElement.comma(), CsmElement.space()), CsmElement.token(ASTParserConstants.LT),
-                CsmElement.sequence(CsmElement.token(ASTParserConstants.GT), CsmElement.space()));
+        return list(ObservableProperty.TYPE_PARAMETERS, CsmElement.sequence(CsmElement.comma(), CsmElement.space()), CsmElement.token(GeneratedJavaParserConstants.LT),
```

Comment:
Not sure whether you clicked that option or not when creating this PR, but since recently one can have other contributors push to the same PR. I.e. @ftomassetti could add this kind of changes right to this PR (it's a checkbox when submitting a PR). Found it quite useful myself.